modules/packages/Sort.chpl

```diff
@@ -1070,6 +1077,7 @@ module QuickSort {
                      minlen=16,
                      comparator:?rec=defaultComparator,
                      start:int = Dom.low, end:int = Dom.high) {
+    import super.InsertionSort;
```

Comment:
Any reason you chose relative over absolute for only this import?

Reply:
Nope.  I introduced the imports in different batches (some showed up from a hello world, others from more extensive sort tests), so probably was just using different approaches in different passes (Michael had me fix at least one other in an earlier review).  I'll change this to `import Sort...` for consistency.